packages/libz/libz.spec

```
@@ -0,0 +1,60 @@
+Name: %{_cross_os}libz
+Version: 1.2.11
+Release: 1%{?dist}
+Summary: Compression and decompression library
+URL: https://www.zlib.net/
+License: zlib and Boost
+
+Source: http://www.zlib.net/zlib-%{version}.tar.gz
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Header files and libraries for Zlib development
```

My convention for these summaries is:
"Summary: Files for development using the $main_package_description"

```suggestion
Summary: Files for development using the library for zlib compression 
```